Player.java

```diff
@@ -1,37 +1,24 @@
-import java.util.*;
-import java.io.*;
-
-public class Player{
-   private String name;
-   private int space;
+public class Player
+{
+   private final String name;
+   private int currentSpace;
    
-   public Player(String n){
-      space = 0;
-      name = n;
-   }
+   public Player(String n)

```

Comment:
ooops... forgot 2 rename the parameter. it should b `name` then @ #7 it will be `this.name = name;`